registry/proxy/proxymanifeststore.go

```diff
@@ -11,15 +11,13 @@ import (
 	"github.com/opencontainers/go-digest"
 )
 
-// todo(richardscothern): from cache control header or config
```

Comment:
I see the original todo mentioned respecting cache-control headers; that still something we want to add at some point?

Reply:
The `TODO` comment actually says:  "...instead of hardcoding an arbitrary chosen constant here, we could *either* pick it from cache headers or we can make move it to config". Given this PR moves the TTL value to config, I think this comment no longer applies.